SPECS/lttng-tools/lttng-tools.spec

```diff
@@ -16,7 +16,7 @@ BuildRequires: elfutils-devel
 BuildRequires: popt-devel
 BuildRequires: userspace-rcu-devel >= 0.8.0
 BuildRequires: lttng-ust-devel >= 2.9.0
-Requires:      lttng-ust = %{version}
+Requires:      lttng-ust >= 2.13.1
```

Comment:
Should these version requirements match, since they come from the same package?

Reply:
There is a newer version of lttng-tools with seemingly useful changes described in the link below. Should I downgrade lttng-tools to match versions?

 https://git.lttng.org/?p=lttng-tools.git;a=blob_plain;f=ChangeLog;hb=0332b18ab18fcca521e94fd623e9e800d44bcae3

Reply:
Sorry, my comment was unclear- I meant to ask if the versions of the build-time requirement on `lttng-ust-devel` and the run-time requirement on `lttng-ust` should match.